storage/innobase/dict/dict0load.cc

```diff
@@ -1483,10 +1483,11 @@ dict_load_column_low(
 		goto err_len;
 	}
 
-	if (!column) {
+	if (!column && table) {
```

Comment:
I just realized that this change causes failures, as unstable means that tests are failing, it's not just a warning.

I'll change back this line and instead of this change, modify the function declaration so that it accepts nulls.

Reply:
Yep, I had a related comment in my review too